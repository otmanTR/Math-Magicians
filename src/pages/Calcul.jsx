import Calculator from '../components/Calculator';

// eslint-disable-next-line import/prefer-default-export
export function Calcul() {
  return (
    <>
      <div className="calcul">
        <h2>Lets do some math!</h2>
        <Calculator />
      </div>
    </>
  );
}
