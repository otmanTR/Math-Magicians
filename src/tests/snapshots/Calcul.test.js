import React from 'react';
import { render } from '@testing-library/react';
import Calcul from '../../pages/Calcul';

describe('Calcul', () => {
  test('should match Calcul snapshot', () => {
    const tree = render(
      <Calcul />,
    );
    expect(tree).toMatchSnapshot();
  });
});
