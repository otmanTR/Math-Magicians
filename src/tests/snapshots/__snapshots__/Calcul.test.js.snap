// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calcul should match Calcul snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calcul"
      >
        <h2>
          Lets do some math!
        </h2>
        <div
          class="container flex"
        >
          <div
            class="child result flex"
          >
            0
          </div>
          <div
            class="child flex"
          >
            <button
              class="button key"
              type="button"
            >
              AC
            </button>
            <button
              class="button key"
              type="button"
            >
              +/-
            </button>
            <button
              class="button key"
              type="button"
            >
              %
            </button>
            <button
              class="button operator"
              type="button"
            >
              รท
            </button>
          </div>
          <div
            class="child flex"
          >
            <button
              class="button key"
              type="button"
            >
              7
            </button>
            <button
              class="button key"
              type="button"
            >
              8
            </button>
            <button
              class="button key"
              type="button"
            >
              9
            </button>
            <button
              class="button operator"
              type="button"
            >
              x
            </button>
          </div>
          <div
            class="child flex"
          >
            <button
              class="button key"
              type="button"
            >
              4
            </button>
            <button
              class="button key"
              type="button"
            >
              5
            </button>
            <button
              class="button key"
              type="button"
            >
              6
            </button>
            <button
              class="button operator"
              type="button"
            >
              -
            </button>
          </div>
          <div
            class="child flex"
          >
            <button
              class="button key"
              type="button"
            >
              1
            </button>
            <button
              class="button key"
              type="button"
            >
              2
            </button>
            <button
              class="button key"
              type="button"
            >
              3
            </button>
            <button
              class="button operator"
              type="button"
            >
              +
            </button>
          </div>
          <div
            class="child flex"
          >
            <button
              class="button key zero"
              type="button"
            >
              0
            </button>
            <button
              class="button key dot"
              type="button"
            >
              .
            </button>
            <button
              class="button operator equal"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calcul"
    >
      <h2>
        Lets do some math!
      </h2>
      <div
        class="container flex"
      >
        <div
          class="child result flex"
        >
          0
        </div>
        <div
          class="child flex"
        >
          <button
            class="button key"
            type="button"
          >
            AC
          </button>
          <button
            class="button key"
            type="button"
          >
            +/-
          </button>
          <button
            class="button key"
            type="button"
          >
            %
          </button>
          <button
            class="button operator"
            type="button"
          >
            รท
          </button>
        </div>
        <div
          class="child flex"
        >
          <button
            class="button key"
            type="button"
          >
            7
          </button>
          <button
            class="button key"
            type="button"
          >
            8
          </button>
          <button
            class="button key"
            type="button"
          >
            9
          </button>
          <button
            class="button operator"
            type="button"
          >
            x
          </button>
        </div>
        <div
          class="child flex"
        >
          <button
            class="button key"
            type="button"
          >
            4
          </button>
          <button
            class="button key"
            type="button"
          >
            5
          </button>
          <button
            class="button key"
            type="button"
          >
            6
          </button>
          <button
            class="button operator"
            type="button"
          >
            -
          </button>
        </div>
        <div
          class="child flex"
        >
          <button
            class="button key"
            type="button"
          >
            1
          </button>
          <button
            class="button key"
            type="button"
          >
            2
          </button>
          <button
            class="button key"
            type="button"
          >
            3
          </button>
          <button
            class="button operator"
            type="button"
          >
            +
          </button>
        </div>
        <div
          class="child flex"
        >
          <button
            class="button key zero"
            type="button"
          >
            0
          </button>
          <button
            class="button key dot"
            type="button"
          >
            .
          </button>
          <button
            class="button operator equal"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
